src/components/Button.jsx

```diff
@@ -8,6 +8,6 @@ export const Button = ({text, isTransparent, isButtonLarge}) => {
   }
 
   return (
-    <button className={`flex align-center bg-blue-500 text-white hover:bg-blue-600 rounded-3xl ${size}`}>{text}</button>
+    <button className={`flex align-center bg-red-500 text-white hover:bg-red-600 rounded-3xl ${size}`}>{text}</button>
```

Comment:
This button should retain a non-red styling option for non-terminating actions (i.e., "submit" vs "cancel"). At the moment, this line is changing every button in the app to red.

Perhaps we can add a prop called `isRed` (or something along those lines) to toggle styling as opposed to hard coding them. And from there we can do:

`const color = isRed ? 'bg-red-500' : 'bg-blue-500';`
`const hoverColor = isRed ? 'hover:bg-red-600' : 'hover:bg-blue-600';`